ci-operator/config/openshift/odo/openshift-odo-main__v4.7.yaml

```diff
@@ -0,0 +1,66 @@
+base_images:
+  cli:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+binary_build_commands: make bin
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/build-root/Dockerfile
+images:
+- context_dir: openshift-ci/build-root/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: oc-bin-image
+resources:
+  '*':
+    requests:
+      cpu: 800m
+      memory: 6Gi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
+tests:
+- as: integration-e2e
+  steps:
+    cluster_profile: aws
+    test:
+    - as: integration-e2e-steps
+      commands: scripts/openshiftci-presubmit-all-tests.sh
+      credentials:
+      - mount_path: /usr/local/ci-secrets/odo-rabbitmq
+        name: odo-rabbitmq
+        namespace: test-credentials
+      env:
+      - default: /usr/local/ci-secrets/odo-rabbitmq/amqpuri
+        name: ODO_RABBITMQ_AMQP_URL
+      from: oc-bin-image
+      resources:
+        requests:
+          cpu: "2"
+          memory: 6Gi
+      timeout: 4h0m0s
+    workflow: ipi-aws
+- as: integration-e2e-periodic
+  cron: 0 */6 * * *
+  steps:
+    cluster_profile: aws
+    test:
+    - as: integration-e2e-periodic-steps
+      commands: scripts/openshiftci-periodic-tests.sh
+      from: oc-bin-image
+      resources:
+        requests:
+          cpu: "2"
+          memory: 6Gi
+      timeout: 4h0m0s
```

Comment:
Why are we not adding timeouts `4h0m0s` for 4.5 periodic job

Reply:
Added, pushing shortly

Reply:
added